### PR TITLE
feat(3000): Relabeled apps to data pipeline, re-ordered nav

### PR DESCRIPTION
**Pull request by corywatilo**
"Apps" are really CDP data connectors, with the exception of [two or three](https://posthog.com/apps).

@tiina303 is working on a bonafide _Data pipeline_ section, but until then, it's worth relabeling Apps as a Data pipeline section. Once Tiina is done, we can just swap to her UI entirely, and re-use `/apps` for the handful of apps we have, and move them somewhere more subtle in the UI.

This also eliminates the third grouping in the nav, leaving us with 1) Data and project, 2) Products, and 3) Utilities

image

Note: I didn't update the "Apps" label in the breadcrumb as I didn't want to affect the existing UI, and it's a relatively minor thing we can address later.

### CODE REVIEW

**Review by thmsobrmlr — APPROVED** [review]
Looks good code-wise and I don't have a strong opinion regarding apps vs connectors.

**Review by tiina303 — CHANGES_REQUESTED** [review]
👍  for the menu change, but let's not rename things to 'connectors' atm

File: frontend/src/scenes/plugins/AppsScene.tsx [hunk 1]
```diff
@@ -56,7 +56,7 @@ export function AppsScene(): JSX.Element | null {
                 activeKey={pluginTab}
                 onChange={(newKey) => setPluginTab(newKey)}
                 tabs={[
-                    { key: PluginTab.Apps, label: 'Apps', content: <AppsTab /> },
+                    { key: PluginTab.Apps, label: 'Connectors', content: <AppsTab /> },
```
[the remark this answers is not in the document]
Comment (corywatilo) [reply frontend/src/scenes/plugins/AppsScene.tsx:59]: What would you suggest calling them in the interim where they're all in a single list?

[the remark this answers is not in the document]
Comment (tiina303) [reply frontend/src/scenes/plugins/AppsScene.tsx:59]: just keep calling them Apps for now

**Review by tiina303 — APPROVED** [review]

**Pull request merged**